virtualizor.py

```diff
@@ -105,6 +105,14 @@ def __init__(self, conf):
         self.conn = libvirt.open('qemu+ssh://root@%s/system' %
                                  conf.target_host)
 
+    def find_emulator(self):
```

Comment:
On 02/19/2015 06:06 AM, Gonéri Le Bouder wrote:
>
> In virtualizor.py
> <https://github.com/enovance/infra-virt/pull/24#discussion_r24979958>:
>
> > @@ -105,6 +105,14 @@ def __init__(self, conf):
> >          self.conn = libvirt.open('qemu+ssh://root@%s/system' %
> >                                   conf.target_host)
> >  
> > +    def find_emulator(self):
>
> I suggest something like that instead, to avoid the serie of if / else:
>
> |def find_emulator(self):
>     for location in ('/usr/bin/qemu-system-x86_64', '/usr/libexec/qemu-kvm'):
>         if self.call('test', '-f', location) == 0:
>             return location
> |
>
That seems a bit odd to use test in python. I would rather see something
like:

def find_emulator(self):
    for location in ('/usr/bin/qemu-system-x86_64',
'/usr/libexec/qemu-kvm'):
        if os.path.isfile(location):
            return location
    return None